packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -88,10 +106,26 @@ const ActionHeader = ({
   onPrevious,
   onNext,
   children,
+  backButtonA11yLabel,
+  nextButtonA11yLabel,
+  prevButtonA11yLabel,
   ...customProps
 }) => {
   const theme = React.useContext(ThemeContext);
 
+  const buttonId = uniqueid();
+  const closeButtonId = `terra-action-header-close-button-${buttonId}`;
+  const maximizeButtonId = `terra-action-header-maximize-button-${buttonId}`;
+  const minimizeButtonId = `terra-action-header-minimize-button-${buttonId}`;
+
+  const visuallyHiddenComponent = (
```

Comment:
Please use the AccessibleValue component, which I think will solve this double-reading problem and was purpose built for this kind of scenario: https://github.com/cerner/terra-framework/blob/c92fd06dc900c13f4c3ce8457066c7db15a5a2f6/packages/terra-time-input/src/_AccessibleValue.jsx#L25

That component was created as part of adding a11yLabels for TimeInput.

Reply:
If you'd prefer, perhaps ship-as-is and then go back and use this component? I think that would be passive and might help you ship faster. Your call. If you'd like to do the iterative approach, tell me so and I'll flip my Request Change to Approved :).

Reply:
Yes I think that would be better, also if the AccessibleValue is a reusable component for these scenarios, I think it would be better to move it to a common folder instead of importing it from time input or adding the same code for using it in another component.

Reply:
Agreed. A third option is to use AccessibleValue as-is, and then move it later - which would still be passive. 

So I take it you will be shipping as-is and then turning around to use AccessibleValue in some form?

Reply:
Yes. For the time being I will be shipping as is and then change to AccessibleValue at a later stage.